.github/workflows/ci.yml

```diff
@@ -28,24 +28,18 @@ jobs:
         include:
         # Use Quarkus JVM images: file registry, AMQP messaging, Command Router with embedded cache
         - device-registry: file
-          commandrouting-mode: command-router
           commandrouting-cache: embedded
-          component-type: quarkus-jvm
           messaging-type: amqp
-        # Use Spring Boot images: jdbc registry, Kafka messaging, Command Router with embedded cache
+        # Use Quarkus JVM images: jdbc registry, Kafka messaging, Command Router with embedded cache
         - device-registry: jdbc
-          commandrouting-mode: command-router
           commandrouting-cache: embedded
-          component-type: spring-boot
           messaging-type: kafka
         # Use Quarkus JVM images: mongodb registry, Kafka messaging, Command Router with data grid
         - device-registry: mongodb
-          commandrouting-mode: command-router
           commandrouting-cache: server
-          component-type: quarkus-jvm
           messaging-type: kafka
 
-    name: "Use ${{ matrix.component-type }} images: ${{ matrix.device-registry }} registry, ${{ matrix.messaging-type }} messaging, ${{ matrix.commandrouting-mode }} with ${{ matrix.commandrouting-cache }} cache"
+    name: "Use Quarkus JVM images: ${{ matrix.device-registry }} registry, ${{ matrix.messaging-type }} messaging, ${{ matrix.commandrouting-mode }} with ${{ matrix.commandrouting-cache }} cache"
```

Comment:
Still `matrix.commandrouting-mode` in this line.

Also the "Quarkus JVM images: " substring can be removed here to shorten the job name.